db/api/v1/db/projects.db.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Projects table database interaction layer. These functions are exported to
+ * provide the ability to query Project specific data.
+ * 
+ * TODO: Insert License
+ */
+
+const utils = require('../utils');
+
+// A query to select projects from the database
+const QUERY_FIND = `
+    SELECT 
+        id,
+        sponsor_id, 
+        name, 
+        description
+    FROM 
+        project 
+    WHERE 
+        TRUE 
+`;
+
+// A query to create a new project in the database
+const QUERY_CREATE = `
+    INSERT INTO project 
+        (sponsor_id, name, description) 
+    VALUES
+        ($1, $2, $3)
```

Comment:
I'm not too familiar with PostGIS, but is this a prepared/pre-compiled statement? If not we should use one to prevent SQL-injection/similar vulnerabilities.

Reply:
From my understanding of the docs (https://node-postgres.com/features/queries) this is how you do a prepared statement.